pom.xml

```diff
@@ -73,11 +59,11 @@
   <distributionManagement>
     <repository>
       <uniqueVersion>false</uniqueVersion>
-      <id>maven.jenkins-ci.org</id>
+      <id>repo.jenkins-ci.org</id>
       <url>https://repo.jenkins-ci.org/releases/</url>
     </repository>
     <snapshotRepository>
-      <id>maven.jenkins-ci.org</id>
+      <id>repo.jenkins-ci.org</id>
```

Comment:
these IDs can't be easily changed, at least not without leaving the old ones in for backwards compatibility.

everyones local credentials will have these IDs defined

Reply:
True, oops 👀